src/render_window.h

```diff
@@ -57,9 +69,42 @@ class RenderWindow : public Nan::ObjectWrap {
 
  private:
   sf::RenderWindow* _window;
+  sf::Mutex* _displayDrawMutex;
```

Comment:
We follow [Google Style Guide](https://google.github.io/styleguide/cppguide.html). So the property name should be `_display_draw_mutex`.